pkg/kiln/docker_registry.go

```diff
@@ -71,8 +73,19 @@ func (imageCreator RegistryImageCreator) DeleteApplication(dockerInfo *DockerInf
 	name := fmt.Sprintf("%s/%s", dockerInfo.RepoName, dockerInfo.ImageName)
 
 	for _, image := range *images {
+		// with GCR Docker Registry, you must delete the tag before the manifest
+		if os.Getenv("DOCKER_PROVIDER") == "gcr" {
```

Comment:
this is reason two for separating implementation of `kiln` with a registry to private & GCR

Thoughts?